ghub.org

```diff
@@ -90,7 +90,7 @@ of sharing a single token between all packages that rely on it.  That
 means that users cannot configure Ghub once and later start using a
 new package without any additional setup.  But Ghub helps with that.
 
-When the user invokes some command defined in some package and that
+When the user invokes some command defined in some package which
```

Comment:
I believe one ought to use `that` when the meaning is lost after removing `that/[,]which ...`. So I guess just drop the `and`?

Reply:
Maybe drop `defined in some package` too? I think it just makes the sentence more confusing.

Reply:
Agreed.